skipper.go

```diff
@@ -1175,6 +1178,23 @@ func listenAndServeQuit(
 	log.Infof("proxy listener on %v", o.Address)
 
 	if srv.TLSConfig != nil {
+		if o.InsecureAddress != "" {
+			log.Infof("insecure listener on %v", o.InsecureAddress)
+
+			go func() {
+				o.Address = o.InsecureAddress
```

Comment:
I do not like that we change `o.Address` here. Maybe we should change `listen` to receive address separately from options.